.releaserc.json

```diff
@@ -0,0 +1,56 @@
+{
+  "branches": [
+    "+([0-9])?(.{+([0-9]),x}).x",
+    "master",
+    "next",
+    "next-major",
+    {
+      "name": "beta",
+      "prerelease": true
+    },
+    {
+      "name": "alpha",
+      "prerelease": true
+    }
+  ],
+  "defaultBranch": "main",
```

Comment:
The default branch of this repo is `master`

Reply:
This prop is not necessary. It was just a test and I forgot to remove it. The relevant prop is "branches".

Removed